draft-ietf-webtrans-http2.md

```diff
@@ -923,6 +993,22 @@ WebTransport streams. The signal is intended for the application using
 WebTransport, which is expected to attempt to gracefully terminate the session
 as soon as possible.
 
+## Capsule Ordering and Reliability
+
+The use of an ordered and reliable transport means that a receiver does not need
+to tolerate capsules that arrive out of order. This differs from QUIC in that a
+receiver is required to treat the arrival of out of order capsules rather than
+being tolerant.
+
+For an intermediary that forwards from an unreliable transport (like
```

Comment:
I would replace "reliable" with "strongly ordered". This makes it sound like you can't rely on QUIC to do its job

Reply:
```suggestion
For an intermediary that forwards from an strongly-ordered transport (like
```